src/common/snippets/src/op/subgraph.cpp

```diff
@@ -427,54 +428,46 @@ void Subgraph::control_flow_transformations(lowered::LinearIR& linear_ir,
     const size_t vector_size = get_generator()->get_target_machine()->get_lanes();
     const int32_t buffer_allocation_rank = static_cast<int32_t>(linear_ir.get_config().m_loop_depth);
 
-    // Ticket: 113666
-    // TODO: Make pass pipeline with backend passes more flexible
-    backend_passes_pre_common.run(linear_ir);
-
-    lowered::pass::PassPipeline common_pipeline;
-    common_pipeline.register_pass<lowered::pass::MarkLoops>(vector_size);
-    common_pipeline.register_pass<lowered::pass::SoftmaxDecomposition>(vector_size);
-    common_pipeline.register_pass<lowered::pass::FuseLoops>();
-    common_pipeline.register_pass<lowered::pass::SplitLoops>();
-    common_pipeline.register_pass<lowered::pass::MoveResultOutOfLoop>();
-    common_pipeline.register_pass<lowered::pass::InsertBuffers>(buffer_allocation_rank);
-    common_pipeline.register_pass<lowered::pass::InsertLoadStore>(vector_size);
-    common_pipeline.register_pass<lowered::pass::MoveScalarToConsumer>();
-    common_pipeline.register_pass<lowered::pass::InsertBroadcastMove>();
-    common_pipeline.register_pass<lowered::pass::LoadMoveBroadcastToBroadcastLoad>();
-
-    common_pipeline.register_pass<lowered::pass::ValidateShapes>();
-
-    common_pipeline.register_pass<lowered::pass::ValidateLoops>();
-    common_pipeline.register_pass<lowered::pass::InitLoops>();
-    common_pipeline.register_pass<lowered::pass::InsertLoops>();
-    common_pipeline.run(linear_ir);
-
-    backend_passes_post_common.run(linear_ir);
-
-    lowered::pass::PassPipeline final_pipeline;
-    final_pipeline.register_pass<lowered::pass::AllocateBuffers>(lowering_result.buffer_scratchpad_size, linear_ir.get_config().m_are_buffers_optimized);
-    final_pipeline.register_pass<lowered::pass::CleanRepeatedDataPointerShifts>();
-    final_pipeline.register_pass<lowered::pass::PropagateLayout>();
-    final_pipeline.register_pass<lowered::pass::CleanupLoopOffsets>();
-    final_pipeline.run(linear_ir);
+    lowered::pass::PassPipeline pipeline(lowered_pass_config ? lowered_pass_config
```

Comment:
Maybe we can set `std::make_shared<lowered::pass::PassConfig>()` as default parameter for `control_flow_transformations` method [as we did](https://github.com/openvinotoolkit/openvino/blob/master/src/common/snippets/include/snippets/op/subgraph.hpp#L147) for `shape_infer_factory` in `convert_body_to_linear_ir`?